qiling/__init__.py

```diff
@@ -1,2 +1,78 @@
 from .core import Qiling
-from .__version__ import __version__
\ No newline at end of file
+from .__version__ import __version__
+
+def __git_info():
```

Comment:
I have a slight concern that variables and functions starting with double underlines are reserved for python internal use.

Reply:
Okay, you are right.